app/controllers/api/v1/users_controller.rb

```diff
@@ -29,6 +29,51 @@ def update
         end
       end
 
+      def exist
+        userData = params[:user]
+        Rails.logger.info "************ exist is = #{userData[:email]}"
+        user = User.where(email: userData[:email]).first
+        Rails.logger.info "************ user is = #{user}"
+        @redirect_path = '/profile' #change this to actually redirect to the social login function
```

Comment:
Do we have a route established for this, so we can use the dynamic `_path` methods here (i.e. `user_profile_path`, etc.), instead of hard coding with a string?

Reply:
I wasn't able to find a _path for this, but I've been thinking about it, and I don't see how a Rails path would work anyway, since the actual location change is taking place in React on the frontend. @redirect path is just getting sent back to the frontend, it's not used to go anywhere in the backend.

Reply:
does react have dynamic path methods?

Reply:
np, the strings will work